internal/identity/identitymanager.go

```diff
@@ -56,7 +56,7 @@ type Manager interface {
 
```

Comment:
I could benefit from some double checking on how I handled the situations with no blockchain plugin. I wasn't sure when to error out or just use the default key/given input.